internal/response/id.go

```diff
@@ -0,0 +1,27 @@
+package response
+
+import (
+	"github.com/ProtonMail/gluon/imap"
+)
+
+type IdResponse struct {
+	imap.ID
+}
+
+func (id *IdResponse) String() string {
+	return "* ID " + id.ID.String()
+}
+
+func (r *IdResponse) Send(session Session) error {
+	return session.WriteResponse(r.String())
+}
+
+type IdNilResponse struct{}
```

Comment:
This type is a valid response, in case we don't wish to send any ID data back. It's not in use currently, but it is part of the extension. I can remove it if you'd like.

Reply:
Removed.